ChaosMod/Effects/db/Misc/MiscSpawnUfo.cpp

```diff
@@ -0,0 +1,334 @@
+/*
+	Author: Kanawanagasaki
+*/
+
+#include <algorithm>
+#include <math.h>
+#include <random>
+#include <stdafx.h>
```

Comment:
```suggestion
#include <stdafx.h>

#include <algorithm>
#include <math.h>
#include <random>
```
stdafx.h should always be the first header. Adding a blank line ensures clangfmt doesn't shuffle it around